source/line.cpp

```diff
@@ -1,10 +1,8 @@
 struct Line{
-    ld a, b, c; // b = 1 linee non verticali, 0 Linee verticali
+    ld a, b, c; // b = 1 non-vertical lines, b = 0 vertical lines
     Line(ld a_, ld b_, ld c_) : a(a_), b(b_), c(c_) {}
     Line(Point<ll>& p1, Point<ll>& p2){
-        if(fabs(p1.x - p2.x) < EPS){
-            a = 1.0; b = 0.0; c = -p1.x;
-        }
+        if(fabs(p1.x - p2.x) < EPS) a = 1.0; b = 0.0; c = -p1.x;
```

Comment:
This doesn't look the same as before, we need commas instead of semicolons - or to wrap it in brackets